CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 - ([#893](https://github.com/demos-europe/demosplan-ui/pull/893)) Add data attributes for E-2-E Test ([@ahmad-demos](https://github.com/ahmad-demos))
 - ([#892](https://github.com/demos-europe/demosplan-ui/pull/892)) DpInput: Add hidden attribute for label ([@gruenbergerdemos](https://github.com/gruenbergerdemos))
```

Comment:
do we still want to persist this entry, then?

Reply:
I left it there, because we still added the hidden attribute to the label prop in DpInput. Do you think it is confusing?

Reply:
you're right, technically DpInput inherits the attr as well. so the changelog entry is ok.